libvcs4j-mapping/src/main/java/de/unibremen/informatik/st/libvcs4j/mapping/Mapping.java

```diff
@@ -187,6 +230,346 @@ public int getOrdinal() {
 	public Result<T> map(final Collection<Mappable<T>> from,
 			final Collection<Mappable<T>> to, final RevisionRange range)
 			throws NullPointerException, IOException {
-		throw new UnsupportedOperationException("Not yet implemented");
+		Validate.noNullElements(from);
```

Comment:
I'm sorry, `noNullElements` is not correct here. According to the documentation `null` values in `from` and `to` are filtered. Didn't notice this line in my first review. However, keep in mind to filter out `null` values later in this method to prevent unwanted behavior.